### ISSUE TITLE
Drupal recipes with versioned postgres get mysql config

### ISSUE THREAD

**Issue by JorisVanEijden** [lando/lando#1454]
TL;DR; `lando/plugins/lando-recipes/types/drupaly/builder.js` uses mysql config if the database is not `'postgres'`. So `postgres:10` or `postgres:9.6` will get mysql config and fail to start.

**Tell us about your setup**

What is your lando version and operating system? 
*v3.0.0-rc.9 on Windows 10 Professional*

**Tell us about your `.lando.yml`**

```yml
name: myproject
recipe: drupal7
config:
  webroot: web
  php: 7.1
  via: apache:2.4
  database: postgres:9.6
  drush: 8
  xdebug: true
```

**Tell us about the command you were running**

```bash
lando start
```

**Tell us about the error you got**

```bash
...
warn: Service database is unhealthy                                                                                             
warn: Run "lando logs -s database"
```

```bash
$ lando logs -s database
Attaching to myproject_database_1
database_1   \| mkdir: cannot create directory ▒/etc/ssh▒: Permission denied
database_1   \| Running command /app-entrypoint.sh /run.sh
database_1   \| Running with exec!
database_1   \|
database_1   \| Welcome to the Bitnami postgresql container
database_1   \| Subscribe to project updates by watching https://github.com/bitnami/bitnami-docker-postgresql
database_1   \| Submit issues and feature requests at https://github.com/bitnami/bitnami-docker-postgresql/issues
database_1   \|
database_1   \| nami    INFO  Initializing postgresql
database_1   \| postgre INFO  ==> No injected postgresql.conf file found. Creating default postgresql.conf file...
database_1   \| postgre INFO  ==> No injected pg_hba.conf file found. Creating default pg_hba.conf file...                                                                                                       
database_1   \| postgre INFO  ==> Deploying PostgreSQL from scratch...
database_1   \| Error executing 'postInstallation': Unable to start com.bitnami.postgresql: pg_ctl: could not start server
database_1   \| Examine the log output.
database_1   \| : pg_ctl: could not start server
database_1   \| Examine the log output.
database_1   \|
```

```bash
$ lando ssh -s database -u root -c 'cat /opt/bitnami/postgresql/logs/postgresql.                                                                                         log'
Starting myproject_database_1 ... done
LOG:  syntax error in file "/opt/bitnami/postgresql/conf/conf.d/lando.conf" line 5, near token "["
LOG:  syntax error in file "/opt/bitnami/postgresql/conf/conf.d/lando.conf" line 98, near token "["
LOG:  syntax error in file "/opt/bitnami/postgresql/conf/conf.d/lando.conf" line 99, near end of line
LOG:  syntax error in file "/opt/bitnami/postgresql/conf/conf.d/lando.conf" line 100, near token "quote-names"
LOG:  syntax error in file "/opt/bitnami/postgresql/conf/conf.d/lando.conf" line 103, near token "["
LOG:  syntax error in file "/opt/bitnami/postgresql/conf/conf.d/lando.conf" line 106, near token "["
FATAL:  configuration file "/opt/bitnami/postgresql/conf/postgresql.conf" contains errors
```

```bash
$ lando ssh -s database -u root -c 'cat /opt/bitnami/postgresql/conf/conf.d/lando.conf'
Starting myproject_database_1 ... done
#
# The MySQL database server configuration file for Lando
#
[mysqld]
#
# * Basic Settings
...
```

**Tell us generally about your bug**

`lando/plugins/lando-recipes/types/drupaly/builder.js` uses mysql config if the database is not `'postgres'`. So `postgres:10` or `postgres:9.6` will get mysql config and fail to start.

**Tell us more**

Could better documentation have prevented this issue? if so what should we change?
Does this bug prevent you from using lando?

**Comment by pirog** [comment]
@JorisVanEijden ahh interesting. Let me try to replicate this.

**Comment by pirog** [comment]
Yup, can definitely replicate this. Great steps here. Should be easy enough to fix!

**Comment by pirog** [comment]
@JorisVanEijden thanks for the report here, fix for this is now in rc10
https://github.com/lando/lando/releases

**Comment by GueGuerreiro** [comment]
The exact same error:

```
database_1         | 2019-07-01 10:29:21.066 GMT [1] LOG:  syntax error in file "/opt/bitnami/postgresql/conf/conf.d/lando.conf" line 5, near token "["
database_1         | 2019-07-01 10:29:21.066 GMT [1] LOG:  syntax error in file "/opt/bitnami/postgresql/conf/conf.d/lando.conf" line 10, near token "default-storage-engine"
database_1         | 2019-07-01 10:29:21.067 GMT [1] LOG:  syntax error in file "/opt/bitnami/postgresql/conf/conf.d/lando.conf" line 98, near token "["
database_1         | 2019-07-01 10:29:21.067 GMT [1] LOG:  syntax error in file "/opt/bitnami/postgresql/conf/conf.d/lando.conf" line 99, near end of line
database_1         | 2019-07-01 10:29:21.067 GMT [1] LOG:  syntax error in file "/opt/bitnami/postgresql/conf/conf.d/lando.conf" line 100, near token "quote-names"
database_1         | 2019-07-01 10:29:21.067 GMT [1] LOG:  syntax error in file "/opt/bitnami/postgresql/conf/conf.d/lando.conf" line 103, near token "["
database_1         | 2019-07-01 10:29:21.067 GMT [1] LOG:  syntax error in file "/opt/bitnami/postgresql/conf/conf.d/lando.conf" line 106, near token "["
database_1         | 2019-07-01 10:29:21.067 GMT [1] FATAL:  configuration file "/opt/bitnami/postgresql/conf/postgresql.conf" contains errors
```

Happens if I configure the .lando.yml file database service like so:

```
name: projectname
recipe: drupal8
config:
  php: '7.2'
  via: nginx
  webroot: .
  xdebug: true
  drush: 8.2.3

services:
  database:
    type: postgres
    portforward: true
    creds:
      database: dbname
```
